src/System.Private.ServiceModel/src/project.json

```diff
@@ -17,7 +17,6 @@
     "System.Linq.Expressions": "4.0.10",
     "System.Linq.Queryable": "4.0.0",
     "System.Net.Http": "4.0.0",
-    "System.Net.Primitives": "4.0.10",
     "System.Net.WebHeaderCollection": "4.0.0",
```

Comment:
The package should be included as part of System.Net.* (at least so it says in project.lock.json) so we shouldn't need to explicitly list it here.